web/sass-files/sass/partials/_sidebar--right.scss

```diff
@@ -8,7 +8,7 @@
     padding: 0;
     background: #fff;
     @include single-transition(transform, 0.5s, ease);
-    right: -320px;
+    right: -400px;
```

Comment:
Yes intentional, I just checked out that the sidebar width of 400 and the right value I gave was 320 so just fixed that.